### ISSUE TITLE
Second time proposal is selected it is not read in VoiceOver

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#27655]
- VSCode Version: Code - Insiders 1.13.0-insider (afe47dc84467b487ce0079ff72f55e7ef4f7ebca, 2017-05-30T10:34:18.253Z)
- OS Version: Darwin x64 15.6.0
- Extensions: none
---

Testing #27315

Steps to Reproduce:

1. Enable VoiceOver
2. Open content assist
3. First proposal is selected and read
4. Close content assist
5. Open content assist with same proposals
6. First proposal is selected, but not read

**Comment by ramya-rao-a** [comment]
Between closing and opening of content assist, does the suggestion list go away or remain?

**Comment by chrmarti** [comment]
It goes away.

**Comment by ramya-rao-a** [comment]
I also see this for suggestion list items that have already been visited

**Comment by JacksonKearl** [comment]
It reads it out every other time it shows up. Interesting.

**Comment by mjbvz** [comment]
I'm still seeing this issue when manually triggering suggestions on 0c2d71e2bb9a020995729ab7d5d5f8e4e897e657

Steps:

1. In JS file, trigger suggestions using ctrl+space on a blank line
1. press esc
1. Trigger suggestions manually again without moving the cursor

The first selected suggestions iis not read out again when you do this

**Comment by isidorn** [comment]
I do not think this is ciritcal, since this was behaving like this for a couple of releases
@JacksonKearl feel free to postpone to August

**Comment by JacksonKearl** [comment]
@mjbvz unable to replicate with NVDA. Is this Voiceover? Can you check if 1.25.1 is the same?

**Comment by JacksonKearl** [comment]
Unable to replicate with Voiceover as well. Could you take another look? This is on 4fa62b246

**Comment by JacksonKearl** [comment]
Note that not reading suggestions when the editor loses then regains focus is a separate bug, tracked at #53719. @sbatten says he was able to verify this bug.

**Comment by joaomoreno** [comment]
@JacksonKearl Moving to August

**Comment by JacksonKearl** [comment]
Note to verifiers: this issue is about the alert not being read twice in the same "context". The issue of alerts not being read when code loses focus then regains it is separate and tracked at #53719.
